internal/operators/ocs/validations.go

```diff
@@ -236,13 +241,17 @@ func (o *operator) setStatusInsufficientResources(nodeInfo *resourceInfo, mode o
 }
 
 // count all disks of drive type ssd or hdd
-func getValidDiskCount(disks []*models.Disk) int {
-	var countDisks int
+func getValidDiskCount(disks []*models.Disk, installationDiskID string) (int64, error) {
+	var countDisks int64
+	var err error
 
 	for _, disk := range disks {
-		if disk.SizeBytes >= conversions.GbToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
+		if (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) && installationDiskID != disk.ID {
+			if disk.SizeBytes < conversions.GbToBytes(ocsMinDiskSize) {
+				err = fmt.Errorf("OCS Invalid Disk Size all the disks present in the cluster should be more than %d GB", ocsMinDiskSize)
+			}
```

Comment:
there is an else missing here.
This should be 
if disk.SizeBytes < conversions.GbToBytes(ocsMinDiskSize) {
				err = fmt.Errorf("OCS Invalid Disk Size all the disks present in the cluster should be more than %d GB", ocsMinDiskSize)
			} else {
                   countDisks++
      }

Reply:
If we have a host with 2 disks(excluding the installation disk), where the size of one disk is less than minDiskSize, then for https://github.com/openshift/assisted-service/blob/master/internal/operators/ocs/ocs_operator.go#L125, then if we include an else then getValidDisk will return 1, and the RAM and CPU requirements will be lesser than expected for the number of disks in the host